apps/cookbook/src/app/showcase/data-table-showcase/data-table-showcase.component.html

```diff
@@ -28,10 +12,39 @@ <h2>Data table with minimal data</h2>
   </cookbook-example-viewer>
 
   <h2>Data table wrapped with Kirby Card</h2>
-  <p>Data table wrapped with card and with selectable rows.</p>
+  <p>Data table wrapped with card, with selectable rows.</p>
   <cookbook-example-viewer [html]="dataTableCardExample.template">
     <cookbook-data-table-card-example #dataTableCardExample></cookbook-data-table-card-example>
   </cookbook-example-viewer>
+
+  <h2>Sortable table</h2>
+  <p>
+    Data table wrapped with card, and sorting on header press. We recommend to visit the
+    <a
+      href="https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/sort#sorting_array_of_objects"
+      class="kirby-external-icon"
+      target="_blank"
+    >
+      MDN docs
+    </a>
+    on sorting, if you're performing client side sorting.
+  </p>
+  <cookbook-example-viewer [html]="dataTableSortableExample.template">
+    <cookbook-data-table-sortable-example
+      #dataTableSortableExample
+    ></cookbook-data-table-sortable-example>
+  </cookbook-example-viewer>
+
+  <p>
+    Please note that the Kirby data table does not do any sorting. The sorting logic must be
```

Comment:
Consider semantics and readability:
```suggestion
    <strong>Please note:</strong> the Kirby data table does not do any data sorting for you. The sorting logic must be
```